### PR TITLE
i156 connect_graph_to_backend

### LINKED ISSUE #156: Connect graph to backend
## Basic Information
- Connect to both user dashboard and team dashboard
- update the total KMs as well for both user and team dashboards

Team Dashboard
- Should filter mileage endpoints by team id and sum the total kms
- displays everyones total km for the day in the graph

User Dashboard
- filter mileage endpoints by user id
- when new mileage is added it should update the graph and total km.

### PR DESCRIPTION
**Pull request by urwrstkn8mare**
## Change Summary

connect graph

### Change Form

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

# Related Issue

- Resolve #156

### CODE REVIEW

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
I will review the code properly tomorrow - I've just been testing and found a few bugs. 

1. When submitting a new mileage the graph and total kms doesnt update without refreshing (i have been adding new entries that are in the past for testing)
2. When you have multiple mileage entries in a day only the first one displays in the graph (they should be combined).

File: server/api/settings.py [hunk 1]
```diff
@@ -12,6 +12,12 @@
 import os
 from pathlib import Path
 
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "root": {"level": "INFO"}
+}
+
```
Comment (ChantelleKerr) [inline server/api/settings.py:20]: Is this needed? Where does the logging occur?

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Some bugs, 

Theres also a bug where the team dashboard doesnt load due to unauthorized access to team endpoint. It happens when I login and go to the team page but when I refresh its good.

File: server/api/settings.py (hunk 1 above)
Comment (urwrstkn8mare) [reply server/api/settings.py:20]: oh that was just for debugging the server code

File: client/src/views/DashboardView.vue [hunk 2]
```diff
@@ -17,7 +17,7 @@
                   <v-icon :icon="method" size="52"></v-icon>
                 </v-col>
                 <v-col>
-                  <v-chip color="green" class="rounded text-h5">{{ tempUserMileage }} KM</v-chip>
+                  <v-chip color="green" class="rounded text-h5">{{ UserMileage }} KM</v-chip>
```
[the remark this answers is not in the document]
Comment (urwrstkn8mare) [reply client/src/views/DashboardView.vue:20]: oh ic

**Comment by urwrstkn8mare** [comment]
@ChantelleKerr i implemented the changes pls have a look to see if it works correctly or not

**Review by ChantelleKerr — APPROVED** [review]
looks good

**Pull request merged**